test/parse.test.js

```diff
@@ -46,6 +46,40 @@ describe('Parse', () => {
     expect(dayjs(time).valueOf()).toBe(moment(time).valueOf())
   })
 
+  it('String RFC 2822, time and zone', () => {
```

Comment:
The added test suit here seems just test Date object rather dayjs here?

Reply:
I just wanted to make that functionality explicit, because right now it is neither documented nor explicitly handled in the tests. Whenever we change our internal implementation we now get feedback whether our assumptions are still valid, right? Will tell you about breaking changes then.